test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -5850,6 +5850,8 @@ def submit(self, obj, script=None, extend=None, submit_dir=None):
                         vncompute = True
                 if obj.script is not None:
                     script = obj.script
+                    if 'Resource_List.vntype' in obj.attributes:
+                        obj.unset_attributes(['Resource_List.vntype'])
```

Comment:
Isn't it a little premature to unset vntype here?  What if the script doesn't have vntype set in it?

Reply:
I have removed this and put inside Job.create_script where I am unsetting "Resource_List.vntype" only if select is specified inside the script body as we can not use it along with select specification. And we are already adding vntype=cray_compute to each select chunk inside create_script code.